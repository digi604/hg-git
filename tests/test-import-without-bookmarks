#!/bin/sh

"$TESTDIR/hghave" git || exit 80

hg init
git init -q

git fast-import --quiet < "$TESTDIR/git-sample-repo"

echo % import without bookmarks
hg gimport
hg log
