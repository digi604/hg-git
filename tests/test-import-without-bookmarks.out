% import without bookmarks
importing Git objects into Hg
at: 0/2
creating bookmarks failed, do you have bookmarks enabled?
changeset:   1:c25ea3d53ce1
tag:         tip
user:        test <test@example.org>
date:        Fri Feb 13 23:31:30 2009 +0000
summary:     add beta

changeset:   0:c50ac8b9444c
user:        test <test@example.org>
date:        Fri Feb 13 23:31:30 2009 +0000
summary:     add alpha

